PencilCase/PencilCase/scripts/Repositories/OrderRepository.ts

```diff
@@ -8,11 +8,15 @@ export class OrderRepository {
         this.db = Application.instance.openDataBase();
     }
 
-    public insert = (order: Order, successCallback?: (transaction: SqlTransaction, resultSet: SqlResultSet) => void, errorCallback?: (transaction: SqlTransaction, sqlError: SqlError) => void) => {
+    public insertSqlStatement = (order: Order): string => {
```

Comment:
Added corresponding function that used to generate sql statements, and made it as public APIs so that it's possible for pages to store sql statements as a collection and execute them as a sql transaction.